src/components/authentication/configureFirebase.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useCallback } from 'react';
+
+const AuthContext = React.createContext(
+    {
+        token: '',
+        isLoggedIn: true,
+        login: (token) => { },
+        logout: () => { },
+    }
+);
+
+export const AuthContextProvider = (props) => {
+    let initialToken;
+
+    const [token, setToken] = useState(initialToken);
+
+    const userIsLoggedIn = !!token;
```

Comment:
why `!!`

Reply:
Because I want to check if the token has been stored data or not:
If yes token is stored then return true;
If no token is stored then return false;